### DISCUSSION TITLE
How can I only download audio (opus audio not mp3)

### DISCUSSION THREAD

**Discussion by yeezylife** [ErrorFlynn/ytdlp-interface#99] [Q&A] [answered]
I looked around and can't seems to find anything in download options that allows me to do this since I don't wanna "convert audio to mp3".

**Comment by ErrorFlynn** [comment, marked as answer]
Sorry I didn't see this sooner. Add this as a custom argument: `-f ba[acodec=opus]`

**Comment by yeezylife** [comment]
You can also right click on a item to select formats.
